ServiceApp/src/org/droidplanner/services/android/core/mission/survey/SurveyImpl.java

```diff
@@ -115,6 +119,26 @@ public static msg_mission_item packSurveyPoint(LatLong point, double altitude) {
         return mavMsg;
     }
 
+    protected msg_mission_item getYawCondition(double angle){
+        return packYawCondition(angle);
+    }
+
+    public static msg_mission_item packYawCondition(double angle){
+        msg_mission_item mavMsg = new msg_mission_item();
+        mavMsg.autocontinue = 1;
+        mavMsg.frame = MAV_FRAME.MAV_FRAME_LOCAL_ENU;
+        mavMsg.command = MAV_CMD.MAV_CMD_CONDITION_YAW;
+        mavMsg.x = 0f;
+        mavMsg.y = 0f;
+        mavMsg.z = 0f;
+        mavMsg.param1 = (float)angle;
+        //@// TODO: 6/24/2016 define angular velocity better
+        mavMsg.param2 = 30;
```

Comment:
@TShapinsky arbitrary value? If not, add a comment describing the origin for the value.